### DISCUSSION TITLE
本地开发的部署文档没有找到，不希望使用docker或者其他平台。

### DISCUSSION THREAD

**Discussion by xaseven** [lobehub/lobehub#2658] [Self Hosting]
我看项目文档挺齐全的，但是没有一个本地开发后的部署文档；我想先自己动手部署到生产环境下看下效果；不是使用docker或者其他平台上；希望能看到一个本地发开后的部署文档，比如说.next文件下前端的文件或者目录是哪些可以部署到nginx上的，server目录下哪些目录或者文件是部署到nodejs环境下的。

**Comment by RubuJam** [comment]
https://github.com/lobehub/lobe-chat/wiki/Intro.zh-CN#本地开发环境设置
按照这个来设置就可以了

https://github.com/lobehub/lobe-chat/wiki/Intro.zh-CN#本地开发环境设置
Just set it up like this

**Reply by xaseven** [reply]
我说的是部署的文档；这个是开发环境下的，现在开发环境没有问题，但是部署到生产环境下的时候非常迷惑

**Reply by arvinxx** [reply]
请参考 Dockerfile 的实现自行尝试，目前没见到谁提过这个事情，感觉算是一个小众需求？ 如果有完整实施路径的话，也欢迎提交 PR 帮忙补充~
